test/functional/admin/api/cms/templates_controller_test.rb

```diff
@@ -24,24 +24,24 @@ def test_show
   def test_show_builtin_partial
     partial = FactoryBot.create(:cms_builtin_partial, provider: @provider)
 
-    get :show, id: partial.id, format: :xml, access_token: @token
+    get :show, params: { id: partial.id, format: :xml, access_token: @token }
 
     assert_response :success
     assert_equal 1, xml_elements_by_key(@response.body, 'builtin_partial').count
     assert_equal 0, xml_elements_by_key(@response.body, 'partial').count
   end
 
   def test_create
-    post :create, section_name: { '0' => 'foooo' }, template: { type: 'page',
-      title: 'About', path: '/about' }, format: :json, access_token: @token
+    post :create, params: { section_name: { '0' => 'foooo' }, type: 'page',
```

Comment:
Is there a reason why you removed `template:` from the params?

Reply:
@jsmadis  Yes test was failing I exactly don't remember the error, but now I re-checked and it worked well so updated the code.